app/src/main/java/at/roteskreuz/stopcorona/model/entities/configuration/ApiConfiguration.kt

```diff
@@ -27,15 +27,18 @@ data class ApiConfiguration(
     @field:Json(name = "diagnostic_questionnaire")
     val diagnosticQuestionnaire: ApiDiagnosticQuestionnaire?,
     @field:Json(name = "page_list")
-    val pageList: ApiPageList?
+    val pageList: ApiPageList?,
+    @field:Json(name = "upload_keys_days")
+    val uploadKeysDays: Int = 2
```

Comment:
This sounds like very interesting constants (was discussed a lot). Could you please move it to `Constants`?

Reply:
updated